tools/tock-register-interface/src/regs.rs

```diff
@@ -293,8 +290,8 @@ impl<R: RegisterLongName> From<LocalRegisterCopy<u32, R>> for u32 {
 /// Specific section of a register.
 #[derive(Copy, Clone)]
 pub struct Field<T: IntLike, R: RegisterLongName> {
-    mask: T,
-    shift: u32,
+    pub mask: T,
```

Comment:
Why are these fields made `pub`?

Reply:
So that other users can extend the register system. For example: https://github.com/brghena/tock-register-example/blob/master/src/cpu_regs.rs#L16

It is annoying that they have to public though. Do you have any suggestions on a better way to do this?

Reply:
Ok, I see. I don't think it possible to leak private items without `getters`.